### DISCUSSION TITLE
GPIO mode

### DISCUSSION THREAD

**Discussion by davek61** [pybricks/support#942] [General]
Hello,
I have discovered that in Spike Prime Python it is possible to control the pin 5 and 5 of the Ports using the following commands

**hub.port.A.mode(hub.port.MODE_GPIO)

a=hub.port.A.dig0
#set the pin 5 as an output pin (pin 6 is dig1) direction(0) sets an input pin
a.direction(1)
#set the pin value to high
a.value(1)**

Are there any plans to implement something similar in Pybricks?

**Comment by dlech** [comment]
We currently don't have anything for this. Feel free to open an issue at https://github.com/pybricks/support for it.
